scrunch/datasets.py

```diff
@@ -3391,26 +3391,31 @@ def obtain_schema(self):
                 schema[a] = svdef
         return schema
 
-    def create_tmp_ds(self, csv_file):
+    def make_tmp_name(self):
+        return "Scrunch-backfill-{}-{}-{}".format(
+            self.dataset.name, self.dataset.id, self.timestamp)
+
+    def create_tmp_project(self, name):
+        return self.root.projects.create(shoji_entity_wrapper({"name": name}))
+
+    def create_tmp_ds(self, name, project, csv_file):
```

Comment:
@jjdelc I think this change might require us to bump this version to 0.19.0. What do you think?